src/components/landing-page/landing-page.controller.ts

```diff
@@ -88,10 +92,20 @@ export class LandingPageController implements angular.IController {
     }
 
     angular.element(this.$window).bind('resize', this.onWindowResize);
+
+    // Now adjust the body area to scroll only if necessary
+    this.onWindowResize();
   };
 
-  private onWindowResize = (event: any) => {
-    angular.element(this.snapElement).css('min-height', this.$window.innerHeight + 'px');
+  private onWindowResize = () => {
+    var width = this.$window.innerWidth;
+    var w: any = $(window)[0];
+
+    if (width > w.patternfly.pfBreakpoints.tablet) {
```

Comment:
Yes, this code is adding enough height to the body area so that when snapping down, the top section (build applications) gets completely scrolled off screen. But, this only happens in non-mobile mode since there is no snapping in mobile mode.